docs/http.rst

```diff
@@ -258,30 +261,32 @@ For the following call definitions:
 Allowed bridges are `gcm` (Google Cloud Messaging), `fcm` (Firebase Cloud
 Messaging), and `apns` (Apple Push Notification system)
 
-   :{token}: The bridge specific public exchange token
+   :{client_token}: The bridge specific public exchange token
 
-Each protocol requires a unique token that addresses the remote application.
+Each protocol may require a unique token that addresses the remote application
 For GCM/FCM, this is the `SenderID` and is pre-negotiated outside of the push
```

Comment:
How about: "this is the 'sender ID' or 'project number', created when you set up a project in the Google Dev Console."